lib/frise/loader.rb

```diff
@@ -36,18 +43,34 @@ def load(config_file, global_vars = {})
     def process_includes(config, at_path, root_config, global_vars)
       return config unless config.class == Hash
 
-      config, defaults_confs = extract_include(config)
-      if defaults_confs.empty?
+      # process $content_include directives
+      config, content_include_confs = extract_content_include(config)
+      unless content_include_confs.empty?
+        raise "A #{@content_include_sym} must not have any sibling key" unless config.empty?
+
+        content = ''
+        content_include_confs.each do |include_conf|
+          extra_vars = (include_conf['vars'] || {}).map { |k, v| [k, root_config.dig(*v.split('.'))] }.to_h
+          extra_consts = include_conf['constants'] || {}
+          symbol_table = merge_at(root_config, at_path, config).merge(global_vars).merge(extra_vars).merge(extra_consts)
```

Comment:
The logic for building the symbol table for `$include` and `$content_include` is very similar. Can we extract this to a method?

Reply:
Sure, changed in 9200e51.